packages/framework/src/Form/Admin/Product/Price/PricesWithCalculatedSellingPricesType.php

```diff
@@ -33,7 +33,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
 
         foreach ($this->pricingGroupFacade->getByDomainId($options['domain_id']) as $pricingGroup) {
             $builder->add((string)$pricingGroup->getId(), MoneyWithCalculatedPriceType::class, [
-                'selling_price' => $sellingPrices !== null ? $sellingPrices[$pricingGroup->getId()]?->getSellingPrice() : null,
+                'selling_price' => $sellingPrices !== null ? $sellingPrices[$pricingGroup->getId()]?->getProductPrice() : null,
```

Comment:
Should be selling_price be renamend to product_price?

Reply:
resolved in 23dde09ec1